docs/developer/first_delivery.md

```diff
@@ -0,0 +1,241 @@
+Now that you've familiarized yourself with what Atomist can do, let's build our first delivery flow.
+
+In this flow we will listen to changes made to your README file. On a push to a repository, Atomist will detect if your README has changed, and print a message. In [Local Mode](/developer/local/), this will print to the terminal feed; in [Team Mode](/developer/team/), we will send the message to a chat channel.
+
+## Prerequisites
+
+Before starting, make sure that you have installed [the atomist CLI](/quick-start/) and created [a blank SDM](/developer/sdm/#creating-an-sdm-project). As well, since we'll be working in TypeScript, it's a good idea to [configure your editor](/developer/typescript/) to work with that toolchain. (Remember to run `npm install` in your SDM's folder.)
+
+You also need a local repository that you don't mind making dummy commits and pushes to. We'll be working with [atomist-seeds/express-es6-rest-api](https://github.com/atomist-seeds/express-es6-rest-api). Fork this repository online; to clone it locally and have Atomist's git hooks already configured, run the following command:
+
+```
+atomist clone https://github.com/<YOUR_GITHUB_HANDLE>/express-es6-rest-api
+```
+
+This clones the project to the folder defined by the `ATOMIST_ROOT` environment variable (which defaults to `$HOME/atomist/projects`).
+
+## Defining your first goal
+
+A [goal](https://docs.atomist.com/developer/goal/) is a function that we want to run during certain events or within specific projects. In this case, our goal is to print a message.
```

Comment:
Actually that's a big simplification. A goal is a really a way to describe a step in your delivery process. It can have zero or more functions (goalExecutors) that actually realise the delivery step. 

I wonder if it is fine to leave this simplified version here and explain it more correctly and fully at a later stage?

Reply:
> I wonder if it is fine to leave this simplified version here

Yes, I think there's a distinct advantage to keeping it streamlined in the beginning.